content/applications/finance/accounting/fiscal_localizations/localizations/egypt.rst

```diff
@@ -0,0 +1,244 @@
+
+.. image:: egypt/eta-api-integration.png
+   :align: center
+   :alt: Configuration of the ETA eInvoicing credentials in Odoo Accounting
+
+.. important::
+   - **Test on your preproduction portal** before starting to issue real invoices on the production
+     ETA portal.
+
+     - **Credentials** for preproduction and production environments are different. Make sure to
+       update them on Odoo when you move from one environment to another.
+
+   - If not done yet, fill out your company details with your company's full address, country, and
+     Tax ID.
+
+.. _egypt/einvoicing-eta-codes:
+
+ETA Codes
+~~~~~~~~~
+
+eInvoicing works with a set of codes provided by the ETA. You can use the `ETA documentation
+<https://sdk.preprod.invoicing.eta.gov.eg/codes/>`_ to code your business attributes.
+
+Most of these codes are handled automatically by Odoo, provided that your :ref:`branches
+<egypt/einvoicing-branches>`, :ref:`customers <egypt/einvoicing-customers>`, and :ref:`products
+<egypt/einvoicing-products>` are correctly configured.
+
+- Company Information:
+
+  - Company Tax ID
+  - | Branch ID
+    | If you have only one branch, use ``0`` as the branch code.
+  - Activity type Code
+
+- Other Information:
+
+  - | Product Codes
+    | Your company's products should be coded and matched with their **GS1** or **EGS** codes.
+  - | Tax Codes
+    | Most of the taxes codes are already configured on Odoo in the :guilabel:`ETA Code (Egypt)`
+      field. We advise you to make sure these codes match your company's taxes.
+
+.. seealso::
+   - `Egyptian eInvoicing & eReceipt SDK - Code Tables
+     <https://sdk.preprod.invoicing.eta.gov.eg/codes/>`_
+   - :doc:`../../taxation/taxes/taxes`
+
+.. _egypt/einvoicing-branches:
+
+Branches
+~~~~~~~~
+
+Create a contact and a journal for each branch of your company and configure its ETA settings.
+
+To do so, go to :menuselection:`Accounting --> Configuration --> Journals`, then click on
+:guilabel:`Create`.
+
+Name the journal according to your company's branch and set the :guilabel:`Type` as
+:guilabel:`Sales`. Next, open the :menuselection:`Advanced Settings` tab and fill out the
+:guilabel:`Egyptian ETA settings` section:
+
+- In the :guilabel:`Branch` field, select the branch's contact or create it.
+- Set the :guilabel:`ETA Activity Code`.
+- Set the :guilabel:`ETA Branch ID` (use ``0`` if you have one branch only).
+
+.. image:: egypt/branch-journal.png
+   :align: center
+   :alt: Sales journal configuration of an Egyptian company's branch
+
+.. important::
+   The contact selected in the :guilabel:`Branch` field must be set as a :guilabel:`Company`
+   (**not** as an :guilabel:`Individual`), and the :guilabel:`Address` and :guilabel:`Tax ID` fields
+   must be filled out.
+
+.. _egypt/einvoicing-customers:
+
+Customers
+~~~~~~~~~
+
+Make sure your customers' contact forms are correctly filled out so your e-invoices are valid:
```

Comment:
In line 102 you spelled it eInvoicing, and on this line, you spelled it e-invoices with a hyphen.

Reply:
Yes, I thought about this too. I need to be consistent, but it seems that the ETA calls it "eInvoicing" and similar, while the main spelling in the dictionary (don't remember which one) spells it e-invoicing. I'll fix this, thanks :)